obal/data/modules/srpm.py

```diff
@@ -83,7 +83,7 @@ def fetch_remote_sources(source_location, source_system, sources_dir):
                 zip_file.extract(zip_info, sources_dir)
 
 
-def build_srpm(module, package, base_dir, sources_dir, scl=None):
+def build_srpm(module, package, base_dir, sources_dir, scl=None, macros=None):
```

Comment:
pylint hates this code:
```
  R: 86, 0: Too many arguments (6/5) (too-many-arguments)
  R: 86, 0: Too many positional arguments (6/5) (too-many-positional-arguments)
```

That said, I don't like that check and would probably just disable it globally.